app/services/images/optimizer.rb

```diff
@@ -1,5 +1,11 @@
 module Images
   module Optimizer
+    def self.call(img_src, service: :cloudinary, **kwargs)
+      return cloudinary(img_src, kwargs) unless imgproxy_enabled?
+
+      public_send(service, img_src, kwargs)
+    end
```

Comment:
I decided to go with @rhymes suggestion to add a guard clause here to override the arg injection. Ideally, I would most likely move the responsibility of this override to a view helper but this achieves what I need with less code and minimal indirection.